specification/managementgroups/resource-manager/Microsoft.Management/stable/2019-11-01/management.JSON

```diff
@@ -0,0 +1,1684 @@
+          "readOnly": true
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "title": "Properties",
+          "$ref": "#/definitions/ManagementGroupInfoProperties"
+        }
+      }
+    },
+    "ManagementGroupInfoProperties": {
+      "description": "The generic properties of a management group.",
+      "type": "object",
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "The AAD Tenant ID associated with the management group. For example, 00000000-0000-0000-0000-000000000000"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the management group."
+        }
+      }
+    },
+    "ManagementGroup": {
+      "description": "The management group details.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the management group.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the resource.  For example, Microsoft.Management/managementGroups",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the management group. For example, 00000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "properties": {
+          "title": "Properties",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ManagementGroupProperties"
+        }
+      }
+    },
+    "ManagementGroupProperties": {
+      "description": "The generic properties of a management group.",
+      "type": "object",
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "The AAD Tenant ID associated with the management group. For example, 00000000-0000-0000-0000-000000000000"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the management group."
+        },
+        "roles": {
+          "description": "The role definitions associated with the management group.",
+          "type": "array",
+          "x-nullable": true,
+          "items": {
+            "type": "string"
+          }
+        },
+        "details": {
+          "title": "Details",
+          "$ref": "#/definitions/ManagementGroupDetails"
+        },
+        "children": {
+          "description": "The list of children.",
+          "x-nullable": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupChildInfo"
+          }
+        },
+        "path": {
+          "description": "The path from the root to the current group.",
```

Comment:
What order is this in?  Top to bottom or bottom to top?  Description should be explicit